godtools/push/action.yaml

```
@@ -41,12 +41,18 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Push tmp docker image
-      if: ${{ inputs.push-tmp-image == 'true' && !startsWith(github.ref, 'refs/tags') }}
+    - name: Push tmp docker images
+      if: ${{ inputs.push-tmp-image == 'true' && !startsWith(github.ref, 'refs/tags') && inputs.tag-latest == 'true' }}
```

can't tmp images be pushed always? like
```suggestion
      if: ${{ inputs.push-tmp-image == 'true' }}
```
so I'd always have images:
- tmp-\<commit-hash>
- tmp-\<branch-name>
- (optional, only on tag): \<tag-name>

btw: before reading source of this action I thought that `tag-latest` means "add tag 'latest' to pushed image" :derp:

---

It was kinda buggy if you would have invalid pair of those flags so I moved it level higher to signify their relation.

Basically `tag-latest` means that you don't point specific tag like `image:this-one` but you have `image`.

There's no sense in retagging `image:this-one` to `image:tmp-dasd` currently.